apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.tsx

```diff
@@ -19,14 +20,17 @@ export const LimitPriceWarning = ({onSubmit, orderData}: LimitPriceWarningProps)
   const {numberLocale} = useLanguage()
   const strings = useStrings()
   const limitPrice = new BigNumber(orderData.limitPrice ?? 0).toFormat(numberLocale)
-  const marketPrice = new BigNumber(orderData.selectedPoolCalculation?.prices.market ?? 0).toFormat(numberLocale)
   const wallet = useSelectedWallet()
   const {closeModal} = useModal()
-
   const tokenToSellInfo = useTokenInfo({wallet, tokenId: orderData.amounts.sell.tokenId})
   const tokenToSellName = tokenToSellInfo.ticker ?? tokenToSellInfo.name ?? '-'
   const tokenToBuyInfo = useTokenInfo({wallet, tokenId: orderData.amounts.buy.tokenId})
   const tokenToBuyName = tokenToBuyInfo.ticker ?? tokenToBuyInfo.name ?? '-'
+  const marketPrice = Quantities.format(
+    orderData.selectedPoolCalculation?.prices.market ?? Quantities.zero,
+    orderData.tokens.priceDenomination,
+    14,
```

Comment:
I don't believe we need here, @jorbuedo ?
```suggestion
```

Reply:
ow yeah, defined to 14, @banklesss please add as a constant on both prices, later we add to the module itself.

Reply:
aa u already did nvm

Reply:
That's the number of decimals we show, probably not needed for the warning.